<ui version="4.0" >
 <class>StyleManager</class>
 <widget class="QWidget" name="StyleManager" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>419</width>
    <height>318</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
    <horstretch>1</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" colspan="2" >
    <widget class="QListWidget" name="styles" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
       <horstretch>1</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item rowspan="3" row="0" column="2" >
    <widget class="QStackedWidget" name="stackedWidget" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Preferred" hsizetype="Expanding" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="currentIndex" >
      <number>1</number>
     </property>
     <widget class="ParagraphGeneral" name="paragraphStylePage" />
     <widget class="CharacterGeneral" name="characterStylePage" />
     <widget class="NewStyleWidget" name="createPage" />
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QPushButton" name="up" >
     <property name="text" >
      <string/>
     </property>
    </widget>
   </item>
   <item row="1" column="1" >
    <widget class="QPushButton" name="down" >
     <property name="text" >
      <string/>
     </property>
    </widget>
   </item>
   <item row="2" column="0" >
    <widget class="QPushButton" name="bNew" >
     <property name="text" >
      <string>&amp;New</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" >
    <widget class="QPushButton" name="bDelete" >
     <property name="text" >
      <string>Delete</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ParagraphGeneral</class>
   <extends>QWidget</extends>
   <header>dialogs/ParagraphGeneral.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>NewStyleWidget</class>
   <extends>QWidget</extends>
   <header>dialogs/NewStyleWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CharacterGeneral</class>
   <extends>QWidget</extends>
   <header>dialogs/CharacterGeneral.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
